source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -591,6 +619,7 @@ FilterDataStatus Filter::onData(ProcessorState& state, Buffer::Instance& data, b
   FilterDataStatus result;
   switch (state.bodyMode()) {
   case ProcessingMode::BUFFERED:
+    // TODO: Moving this code into a method handleDataBuffered().
```

Comment:
TODO(your-github-name): Refactor this code for readability and testability, using a method per case.

You can alternatively use `TODO(#github-issue-number)`

Reply:
done